docs/2-advanced/01-configuration.md

```diff
@@ -174,12 +174,16 @@ For example, to force the first day of week to Sunday, no matter the locale:
 }
 ```
 
-Setting     |             Values              |    Default Value    |                  Description
--------------- | ------------------------------- | ------------------- | ----------------------------------------------
-firstDayOfWeek | 0 (Sunday) through 6 (Saturday) | *Depends on locale* | When set, overrides the locale's default value
-
-**Note:** Currently, only `firstDayOfWeek` is supported. More settings may be added in the future.
+Setting                       |             Values              |    Default Value    |                  Description
+------------------------------| ------------------------------- | ------------------- | ----------------------------------------------
+firstDayOfWeek                | 0 (Sunday) through 6 (Saturday) | *Depends on locale* | When set, overrides the locale's default value
+legacyDateCalendarCustomizing | {<br />dateFormat: string,<br />islamicMonthStart: string,<br />gregDate: string<br />} | []                  | When set, adds customizing data for Islamic calendar support
+legacyDateFormat              | 1\|2\|3\|4\|5\|6\|7\|8\|9\|A\|B\|C         | undefined           | When set, specifies legacy ABAP date format
```

Comment:
I think we need to avoid using ABAP here and below, although ABAP apps could be the only use-cases. Let's stick to "legacy" and explain what the APIs do and when exactly one can be interested in using them